src/sources/dnstap/mod.rs

```diff
@@ -30,8 +30,9 @@ use vector_core::config::{LegacyKey, LogNamespace};
 #[configurable_component(source("dnstap"))]
 #[derive(Clone, Debug)]
 pub struct DnstapConfig {
-    /// Maximum length, in bytes, that a frame can be.
+    /// Max DNSTAP frame length that the dnstap source will accept.
```

Comment:
```suggestion
Maximum DNSTAP frame length that the source will accept.

If any frame is longer than this, it will be discarded.
```

Reply:
Maybe the wording of the second sentence could use some wordsmithing/changes, but:

- "maximum" instead of "max"
- we should generally explain what happens in negative behavior scenarios, even if it's obvious to us i.e. does an oversized frame actually break/stop the source? or does it just discard them and keep processing? etc